README.md

```diff
@@ -33,52 +37,55 @@ in your `project/AppDependencies.scala` file. For example,
     libraryDependencies += "uk.gov.hmrc" %% "play-frontend-hmrc" % "x.y.z-play-27"
     ```
     
-    The library is cross-compiled for Play 2.6 and 2.7.
+    The library is cross-compiled for Play 2.6 and 2.7. Note, because this library transitively depends on play-frontend-govuk
+    there is no need to add this as a separate dependency. If it is already included, it can be removed.
 
-    **As of v0.35.0 (January 2021), this library will no be cross-compiled against Play 2.5.**
-
-1.  Import the required styles in your `app/assets/stylesheets/application.scss` file:
-    ```scss
-    $govuk-assets-path: "/<route-to-your-service>/assets/lib/govuk-frontend/govuk/assets/";
-    $hmrc-assets-path: "/<route-to-your-service>/assets/lib/hmrc-frontend/hmrc/";
-    
-    @import "lib/govuk-frontend/govuk/all";
-    @import "lib/hmrc-frontend/hmrc/all";
-    ```
-
-1.  Add routes for hmrc-frontend assets in `conf/app.routes`:
+1.  Add routes for hmrc-frontend and govuk-frontend assets in `conf/app.routes`:
     ```scala
     ->         /govuk-frontend                     govuk.Routes
     ->         /hmrc-frontend                      hmrcfrontend.Routes
     ```
 
-1. Ensure you have the correct routing for all other static assets including the compiled Javascript and images provided 
-by the hmrc-frontend webjar:
-    ```
-    GET        /assets/*file                        controllers.Assets.versioned(path = "/public", file: Asset)
-    ```
+1. Ensure you have your service name defined in your messages files. For example,
+    ```scala
+    service.name = Any tax service
+    ``` 
+
+1. Create a layout template using govukLayout that you can use in all of your service pages. For example,
 
-1. Update your scripts template to include the hmrc-frontend javascript assets:
     ```scala
-    @this()
+    @import uk.gov.hmrc.hmrcfrontend.views.html.helpers.{HmrcStandardFooter, HmrcStandardHeader, HmrcHead, HmrcScripts}
+
+    @this(
+        govukLayout: GovukLayout,
+        hmrcHead: HmrcHead,
+        hmrcStandardHeader: HmrcStandardHeader,
+        hmrcStandardFooter: HmrcStandardFooter,
+        hmrcScripts: HmrcScripts
+    )
+    
+    @(pageTitle: String)(contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
     
-    @()
```

Comment:
Question: is this worth leaving in as a "Prior to version xx.xx.xx" note? Or do you think that would be confusing?

Reply:
I think it might have the potential to misdirect/be confusing. Teams using older versions can always browse to the previous tag if they want to see the documentation relating to prior versions.